From f6df6a1f0ccacf3b7fd0c4baebc47390e16cbe99 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Michael=20La=C3=9F?= <lass@mail.uni-paderborn.de>
Date: Fri, 26 Jul 2013 14:45:05 +0200
Subject: [PATCH 3/3] Enable encryption after starting the client.

Although the documentation says that encryption is enabled by default it is
not. Enable it by running "fs setcrypt on" after starting the client.

Change-Id: Id4d1920e9dcfa950c7a04b34c0773ee7efdf1b27
---
 src/packaging/RedHat/openafs-client.service | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/packaging/RedHat/openafs-client.service b/src/packaging/RedHat/openafs-client.service
index ac479e7..01f156f 100644
--- a/src/packaging/RedHat/openafs-client.service
+++ b/src/packaging/RedHat/openafs-client.service
@@ -7,6 +7,7 @@ Type=forking
 EnvironmentFile=/etc/conf.d/openafs
 ExecStartPre=/usr/bin/modprobe openafs
 ExecStart=/usr/bin/afsd $AFSD_ARGS
+ExecStartPost=/usr/bin/fs setcrypt on
 ExecStop=/usr/bin/umount /afs
 ExecStop=/usr/bin/afsd -shutdown
 ExecStop=/usr/bin/rmmod openafs
-- 
1.8.3.3

